src/atmos_param/cloud_simple/lcl.F90

```diff
@@ -0,0 +1,920 @@
+! The original code is from http://romps.berkeley.edu/papers/pubdata/2016/lcl/lcl.f90
```

Comment:
Can you check that we can reproduce this code in here? I.e. take a look at the license Romps uses. I also see that there are other functions with different licenses in here. Can you make sure they are all open enough for us to use too please?

Given it was taken directly from Romps (which is a very trustworthy source), I propose we don't make any changes)